examples/next-ai-rsc/package.json

```diff
@@ -19,7 +19,7 @@
     "@radix-ui/react-tooltip": "^1.0.7",
     "@vercel/analytics": "^1.2.2",
     "@vercel/kv": "^1.0.1",
-    "ai": "3.0.4",
+    "ai": "latest",
```

Comment:
why is this necessary?

Reply:
This avoids that we need to bump the version number here manually whenever we do an AI package release.

Reply:
shouldn't this happen automatically as it does for all other packages?